bootstrap.ml

```diff
@@ -30,7 +30,7 @@ end
 (* Directories with library names *)
 let dirs =
   [ ("vendor/boot"      , None)
-  ; ("vendor/usexp/src" , Some "Usexp")
+  ; ("src//usexp/src"   , Some "Usexp")
```

Comment:
`//` --> `/`.  You can also remove `src/usexp/LICENSE.md` (it's the same as the toplevel license file) and move the files from `src/usexp/src` to `src/usexp` directly

Reply:
I'm in the process of doing that! :)